diem-unified-charter.md

```diff
@@ -1,49 +1,93 @@
-digital (e.g., a web server). They
-differ in the initial interaction between the asset and the validator. Digital
-Emblems can fall into multiple types (e.g., self-signed, attested, etc.) that will
-affect how a validator interprets and trusts the assertions made within the emblem.
-The relationship between the asset and the validator will influence requirements on how
-validators retrieve digital emblems for the asset. These three dimensions will be key
-drivers in the development of a digital emblem architecture.
-
-# Initial Scope
-
-The working group will focus its initial work on assets that are discovered by use
-of digital communication as the primary interaction method that have the
-unique challenge of the validator remaining unknown to the operator of the asset.
-Other use cases for future consideration may also cover assets that lack this requirement
-or even require separate digital discovery. The WG will ensure maximum reuse of work
-for emblem representation and binding. A discovery
-mechanism for the initial work will only be specified by this group after the initial emblem
-binding protocol is completed (see the Deliverables section below).”
-
-The working group is tasked with developing: 1) use cases and requirements related to the
-initial scope, 2) an architecture that captures the relationships between entities utilizing
-digital emblems, 3) a specification on binding digital emblems to assets, and 4) a specification
-for discovery mechanisms supported by the initial scope.
-
-The working group will liaise with appropriate organizations and relevant IETF WGs when
-defining emblem formats, validation, and discovery mechanisms to facilitate re-use of existing
-protocols and capabilities.
-
-## Deliverables
-
-The DIEM WG will work on the following milestones for the defined scope:
-
-1. Use cases and requirements for the initial target scenarios (Informational)
-2. Architecture document containing a taxonomy and overall information flows (Informational)
-3. A protocol specification describing the binding of emblems to assets (Proposed Standard or Experimental)
-4. A protocol specification describing the emblem discovery mechanism for the initial use cases (Proposed Standard
-or Experimental)
+## Background
+
+An emblem is a device, symbol, or figure adopted and used as an identifying mark.
+In culture, emblems such as a flag, badge or coat of arms communicate group identity.
+In speech, emblems are specific nonverbal gestures or signals that have a direct verbal translation and are widely understood within a particular culture or community.
+The emblems of the International Committee of the Red Cross (ICRC)  are symbols of protection under the Geneva Conventions and are to be worn by all medical and humanitarian personnel and also displayed on their vehicles and buildings while they are in an active warzone, and all military forces operating in an active warzone must not attack entities displaying these emblems.
```

Comment:
```suggestion
The emblems of the International Committee of the Red Cross (ICRC) are symbols of protection under the Geneva Conventions.
```

Reply:
@mstojens I opted to shorten to eliminate the partial definition, hopefully this would address both of your comments.